### PR TITLE
locfig fix (new variables: fieldid, spicyid, name)

### PR DESCRIPTION
**Pull request by aaronrogue**
fixes and uncomments the code to present the locfig as part of the graph

This code requires you have a folder in your main directory called /figures/ with the location images of each of your sources in it. To do this automatically, run the Figure Getter notebook I also uploaded. 

It also requires, in order to pull the images, that you have the following variables defined somewhere prior:
    fieldid = 'G328' (ex. - whatever your region is)
    spicyid = 31415 (ex. - whatever source you're looking at)
    name = 'btingle' (however your name appears in your directory, aka /home/yourname)

I also tweaked it so that it doesn't attempt to display the chi2limit (as in chi2limit, not deltachi2limit) since I suspect that we each have that defined in different ways or not at all. If anyone wants to reintroduce the chi2 label that'd be welcome.

### CODE REVIEW

File: plot_fit.py [hunk 1]
```diff
@@ -78,6 +78,12 @@ def plot_fit(bestfits_source, geometries_selection,
              alpha_allmodels=0.1,
             ):
 
+    # This code will break if you don't have the following variables defined and passed into the function:
+    # fieldid = 'G328' (ex. - whatever your region is)
+    # spicyid = 31415 (ex. - whatever source you're looking at)
+    # name = 'btingle' (however your name appears in your directory, aka /home/yourname)
+    # They are used to pull the location image.
+
```
Comment (keflavich) [inline plot_fit.py:86]: these are called "positional arguments".

The standard is to document them as you have, but in a "docstring", e.g., like this:
```suggestion
    """
    Parameters
    ----------
    fieldid : string
         'G328' (ex. - whatever your region is)
    """
```
etc.

**Review by keflavich — APPROVED** [review]

**Comment by keflavich** [comment]
This is good.  

The path specification is a bit inflexible, though, and so I will suggest some changes.

File: plot_fit.py [hunk 2]
```diff
@@ -69,15 +69,25 @@ def binsfunction(param, kind, binsnum, deltachi2lim, geometries, bestfits, massn
     return bins
 
 def plot_fit(bestfits_source, geometries_selection,
-        deltachi2limit,
+        deltachi2limit, fieldid, spicyid, name,
```
Comment (keflavich) [inline plot_fit.py:72]:
```suggestion
        deltachi2limit, fieldid, spicyid,
        figurepath=os.path.expanduser('~/figures'),
```

File: plot_fit.py [hunk 3]
```diff
@@ -253,15 +263,13 @@ def plot_fit(bestfits_source, geometries_selection,
...
-    # locfig = np.flipud(locfig)
-
-    # ax9 = basefig.add_subplot(gs[0, 0])
-    # ax9.imshow(locfig)
-    # ttl = ax9.set_title(f'\n{fieldid}  |  SPICY {spicyid}\n', fontsize=25)
-    # ttl.set_position([.5, 1])
-    # #ax9.axis([90,630,90,630])
-    # ax9.axis([170,550,170,550])
-    # ax9.axis('off')
+    locfig = mpimg.imread(f'/home/{name}/figures/{fieldid}_{spicyid}.png')
```
Comment (keflavich) [inline plot_fit.py:266]:
```suggestion
    locfig = mpimg.imread(f'{figurepath}/{fieldid}_{spicyid}.png')
```

File: plot_fit.py (hunk 2 above)
Comment (keflavich) [reply plot_fit.py:72]: with this change, you don't need to specify your name - it will default to your home directory

**Review by keflavich — CHANGES_REQUESTED** [review]
I have an idea to make the save path a little more flexible.

We don't necessarily want to save files into our home directories, after all

File: plot_fit.py (hunk 2 above)
Comment (aaronrogue) [reply plot_fit.py:72]: that's great, now i know you can do that

**Pull request merged**